src/Hosting/Abstractions/src/EnvironmentName.cs

```diff
@@ -6,10 +6,11 @@ namespace Microsoft.Extensions.Hosting
     /// <summary>
     /// Commonly used environment names.
     /// </summary>
+    [System.Obsolete("Use Environments instead.", error: false)]
```

Comment:
Talked to @Tratcher on Teams. This needs to be updated to fit our obsoletion guidelines:

https://github.com/aspnet/AspNetCore/wiki/Engineering-guidelines#breaking-changes

Reply:
https://github.com/aspnet/Extensions/issues/1148